src/conv/invokers/impl_gemm_dynamic.cpp

```diff
@@ -107,6 +214,30 @@ InvokerFactory MakeImplGemmDynamicDataInvokerFactory(const ConvolutionContext& c
             };
         };
     }
+    else if(ctx.direction.IsBackwardWrW())
+    {
+        return [ctx](const std::vector<Kernel>& kernels) {
+            return [=](const Handle& handle, const boost::any& primitive_parameters) {
+                const auto data_ctx = boost::any_cast<conv::WrWInvokeParams>(primitive_parameters);
+                const auto& tensors = data_ctx.tensors;
+                const auto& workSpace = data_ctx.workSpace;
```

Comment:
clang-tidy raise a warning here:
/var/jenkins/workspace/MLLibs_MIOpen_wrw_with_reduction/src/include/miopen/conv/wrw_invoke_params.hpp:34:8: error: constructor does not initialize these fields: workSpace, workSpaceSize [hicpp-member-init,-warnings-as-errors]

struct WrWInvokeParams

       ^

src/CMakeFiles/tidy-target-MIOpen-solver_conv_asm_implicit_gemm_wrw_v4r1_dynamic_cpp.dir/build.make:57: recipe for target 'src/CMakeFiles/tidy-target-MIOpen-solver_conv_asm_implicit_gemm_wrw_v4r1_dynamic_cpp' failed

But I found that workspace and worksapcesize is initialized here: `ConvolutionUserBuffers bufs(workSpace, workSpaceSize);` in file "convolutionocl.cpp" line 3510-3520. 
@asroy @TejashShah @carlushuang Do you have any suggestions on how to fix this non-initialization error?
Besides, I'll refine the invoker code as #272 . But I think this error probably be the same after refactored.